### PR TITLE
Refs [TASK][27694] [trainer] view subject details

### PR DESCRIPTION
**Pull request by anhnt-1044**
https://edu-redmine.sun-asterisk.vn/issues/27694
Screenshot from 2020-09-05 00-19-14

### CODE REVIEW

**Comment by anhnt-1044** [comment]
ready

**Comment by anhnt-1044** [comment]
ready

**Comment by anhnt-1044** [comment]
ready

File: app/javascript/packs/trainer/subject.js [hunk 1]
```diff
@@ -35,12 +35,12 @@ export function deleteSubject() {
...
 }
 
-export function customAlert(type) {
+export function customAlert(type, action = "delete") {
   let prefix = type
   Swal.fire({
-    title: I18n.t("js.trainer.subject.res." + prefix + "_title"),
-    text: I18n.t("js.trainer.subject.res." + prefix + "_text"),
-    icon: I18n.t("js.trainer.subject.res." + prefix + "_class"),
+    title: I18n.t("js.trainer.subject.res." + action + "." + prefix + "_title"),
```
Comment (ngocvt-0484) [inline app/javascript/packs/trainer/subject.js]: tạo 1 biến chứa string này đi e 
"js.trainer.subject.res." + action + "." + prefix

**Comment by anhnt-1044** [comment]
ready

**Pull request merged**